chebi-update/src/main/java/org/reactome/release/chebiupdate/Main.java

```diff
@@ -27,36 +21,8 @@ public static void main(String[] args) throws SQLException, Exception
 		
 		Properties props = new Properties();
 		props.load(new FileInputStream(pathToResources));
-		MySQLAdaptor adaptor = getMySQLAdaptorFromProperties(props);
-		boolean testMode = new Boolean(props.getProperty("testMode", "true"));
-		if (!testMode)
-		{
-			logger.info("Test mode is OFF - database will be updated!");
-		}
-		else
-		{
-			logger.info("Test mode is ON - no database changes will be made.");
-		}
-		long personID = new Long(props.getProperty("person.id"));
-		ChebiUpdater chebiUpdater = new ChebiUpdater(adaptor, testMode, personID);
-		
-		logger.info("Pre-update duplicate check:");
-		chebiUpdater.checkForDuplicates();
-		chebiUpdater.updateChebiReferenceMolecules();
-		logger.info("Post-update duplicate check:");
-		chebiUpdater.checkForDuplicates();
-	}
 
-	private static MySQLAdaptor getMySQLAdaptorFromProperties(Properties props) throws IOException, FileNotFoundException, SQLException
-	{
-		
-		String dbHost = props.getProperty("db.host", "localhost");
-		String dbUser = props.getProperty("db.user");
-		String dbPassword = props.getProperty("db.password");
-		String dbName = props.getProperty("db.name");
-		int dbPort = new Integer(props.getProperty("db.port", "3306"));
-		
-		MySQLAdaptor adaptor = new MySQLAdaptor(dbHost, dbName, dbUser, dbPassword, dbPort);
-		return adaptor;
+		ChebiUpdateStep chebiUpdateStep = new ChebiUpdateStep();
+		chebiUpdateStep.executeStep(props);
```

Comment:
For my own curiousity, why is it better to name a variable for the class rather than just call the classes function directly: example: `ChebiUpdateStep.executeStep(props);`

Reply:
`executeStep` is not a static method. An instance of `ReleaseStep` is required.
As for the name: yes, I could have just named that variable *step*, or *chebiStep*... but I went with *chebiUpdateStep*, I guess to make it very clear what kind of step it is.